.github/actions/h1-cli/action.yml

```diff
@@ -0,0 +1,20 @@
+name: Set up HyperOne CLI on GitHub Actions
+author: Adam Dobrawy
+
+inputs:
+  version:
+    description: |-
+      Version of the CLI to install. If unspecified or set to "latest",
+      the latest available CLI version will be installed. Example: "1.10".
+    default: latest
+    required: false
+  scope:
+    description: |-
+      Scope of the CLI to install. If unspecified "h1" CLI version will 
+      be installed. Example: "h1", "rbx".
+    default: h1
+    required: false
```

Comment:
Czy ta akcja nie powinna odpowiadać również za konfiguracje danych autoryzacyjnych? Wyobrażam sobie, że ona raz się zaloguje i zapamięta akcje, aby każde kolejne operacje nie musiały ręcznie określać danych.
Wtedy deploy by mogła wyglądać następująco:
```yml
  deploy:
    runs-on: ubuntu-latest
    if: github.ref == 'refs/heads/dev'
    needs:
      - test
    steps:
    - uses: actions/checkout@v1
    - name: Setup H1-CLI
      uses: ./.github/actions/h1-cli
      with:
        access_token: ${{ secrets.HYPERONE_TOKEN }}
        early_adoptes: 'true'
        project: '587707232a57b6fd80f04bc1' 
    - uses: webfactory/ssh-agent@v0.3.0
      with:
        ssh-private-key: ${{ secrets.SSH_PRIVATE_KEY }}
    - name: Inject website ssh-hostkey
      run: echo '${{ secrets.SSH_KNOWN_HOSTS }}' >> ~/.ssh/known_hosts;
    - name: Deploy update
      run: make deploy_backend
```
To drobna zmiana, ale gdy wykonuje się troszkę więcej operacji to mogło by to byc pomocne.

Reply:
W nowej wersji CLI, gdzie będziemy mieli "passport-file" (podobnie jak gcloud) będziemy musieli to w taki sposób zrealizować. W sumie już teraz mogę dopisać, ale zakładamy, że w CLI v2 będą breaking changes.